NEWS.md

```diff
@@ -1,46 +1,52 @@
 # forcats (development version)
 
-* `first2()`, a `fct_reorder2()` helper function, sorts `.y` by the first value of `.x` (@jtr13).
+* `first2()`, a `fct_reorder2()` helper function, sorts `.y` by the first value
+  of `.x` (@jtr13).
+
+* `fct_reordern()` is a new function to order based on an arbitrary number of
+  values (@billdenney, #16)
 
 # forcats 0.4.0
 
 ## New features
 
-* `fct_collapse()` gains a `group_other` argument to allow you to group all 
+* `fct_collapse()` gains a `group_other` argument to allow you to group all
```

Comment:
You seem to have changed multiple lines in this file. Can you please revert?

Reply:
Sure.  As an FYI, the changes were made by my default emacs settings which remove blank space at the end of lines, and then I tried to make the style match the tidyverse style guide on line wrapping (https://style.tidyverse.org/news.html#bullets).

Reply:
As a general rule, when working with someone else's code, you should suppress all the automated formatting that you might usually do.

Reply:
Fair enough.